src/components/views/rooms/RoomList.tsx

```diff
@@ -598,6 +610,12 @@ export default class RoomList extends React.PureComponent<IProps, IState> {
             }
         }
 
+        if (!config.showExplorePublicRoom) {
```

Comment:
This looks odd, why is this prompt related to not showing the explore public rooms?

Reply:
'use_custom_text_label_on_left_menu' has been used now with new commits pushed. Text had mentioned about explore public room which might be why I thought lets use that flag but with your suggestion, i have used above flag. With this flag any new text that is passed through config can be used. I wish matrix had provided that in beginning instead of hardcoding text on left menu panel but with my changes it re assigns text value which will align with any client's text requirement. Also documented in feature-communicate-md.